src/Tribe/Main.php

```diff
@@ -19,7 +19,7 @@ class Tribe__Main {
 	const OPTIONNAME        = 'tribe_events_calendar_options';
 	const OPTIONNAMENETWORK = 'tribe_events_calendar_network_options';
 	const FEED_URL          = 'https://theeventscalendar.com/feed/';
-	const VERSION           = '6.4.0';
+	const VERSION           = '7.0.0';
```

Comment:
Are we sure about this version update ? Since its a feature i would expect just 6.5.0

Reply:
No - this was to ensure it was ahead of anything else we were working on.
It should get changed before it gets merged into a release branch.